make-rules/depend.mk

```diff
@@ -1,32 +1,19 @@
 #
-# CDDL HEADER START
+# This file and its contents are supplied under the terms of the
```

Comment:
I deleted the file and recreated it from scratch. That's why it is shown like this.